examples/mining_ar.py

```diff
@@ -0,0 +1,128 @@
+    'bold_yellow': '\033[1;33m',
+    'bold_blue': '\033[1;34m',
+    'green': '\033[32m',
+    'yellow': '\033[33m',
+    'default': '\033[0m',
+    'blue_bg': '\033[1;46m',
+    'default_bg': '\033[1;49m'
+}
+
+def print_table(table, title = None):
+    if title is not None:
+        print(title)
+
+    print(tabulate(table, tablefmt='pipe', stralign='center'))
+
+
+def print_ars(ars):
+    print('Total count of ARs:', len(ars))
+    print('First 10 ARs:')
+    for ar in ars[:10]:
+        print(COLOR_CODES['bold_green' if ar.confidence > 0.9 else \
+                          'bold_yellow' if ar.confidence > 0.3 else 'bold_red'],
+              '{:1.2f}'.format(ar.confidence), COLOR_CODES['default'], end='\t')
+        print(ar.left, '->', ar.right)
+
+
+def print_itemnames(itemnames):
+    print(f'{COLOR_CODES["blue_bg"]}Total number of items:'
+          f'{COLOR_CODES["default_bg"]} {len(itemnames)}')
+    print(*itemnames, sep='\n')
+
+
+def scenario_tabular():
+    algo = desbordante.ar.algorithms.Default()
+    algo.load_data(table=(TABLE_TABULAR, ',', True), input_format='tabular')
+    algo.execute(minconf=1)
+    table = pandas.read_csv(TABLE_TABULAR, header=None)
+
+    print("As the first example, let's look at the dataset containing receipts "
+          'from some supermarket using input_format="tabular". In this '
+          'format, each table row lists all items participating in the same '
+          'transaction. Note that in this table some rows may be shorter than others.\n')
+    print_table(table)
+    print("\nLet's see the first 10 association rules (ARs) that are present "
+          'in the dataset with minconf=1. If no minsup is specified, '
+          'the default value of minsup=0 is used.\n')
+    print_ars(algo.get_ars())
+    print("\n['Eggs'] -> ['Milk'] with confidence 1 means that whenever eggs "
+          'are found in the receipt, milk will '
+          f'{COLOR_CODES["green"]}always{COLOR_CODES["default"]} '
+          'be present as well. The same holds true for all other rules with '
+          f'{COLOR_CODES["bold_green"]}confidence 1{COLOR_CODES["default"]}.')
+
+    print("\n\nNow, let's examine the same dataset with "
+          f'{COLOR_CODES["yellow"]}minconf=0.7{COLOR_CODES["default"]}')
+    algo.execute(minconf=0.7)
+    print_ars(algo.get_ars())
+    print("\n['Milk'] -> ['Eggs'] with confidence 0.75 means that when milk "
+          'is found in the receipt, the chance of eggs being '
+          'present amounts to 75%. So, customers are '
+          f'{COLOR_CODES["bold_yellow"]}likely{COLOR_CODES["default"]} to buy eggs '
+          f'with milk, but {COLOR_CODES["bold_red"]}not always.{COLOR_CODES["default"]}')
+
+    print('\nLet us turn to the next issue. You can observe that there are too many '
+          'association rules found in this small dataset. This happens because we did not '
+          'set up the support value. Due to this fact, the system discovers '
```

Comment:
"support value" can be highlighted as it is a term that was not yet introduced. There's also no explanation that would directly follow said term

"set up the so called support value. This is [support value's definition]"

Adding this would also require you to change the next sentence to accommodate for the change in flow of text here